README.md

```diff
@@ -77,6 +77,12 @@ public enum MyHandler implements RestrictionHandler {
     public boolean canUltimine(Player player) {
         return player.getMainHandItem().getItem() instanceof SomeCustomItem;
     }
+
+    @Override
+    public String ultimineBlockReason()
+    {
```

Comment:
For consistency with the rest of the project, please keep opening braces `{` on the same line as the statement